packages/react/src/components/pickers/Suggestions/Suggestions.tsx

```diff
@@ -356,6 +356,7 @@ export class Suggestions<T> extends React.Component<ISuggestionsProps<T>, ISugge
       suggestionsContainerAriaLabel,
       suggestionsHeaderText,
       suggestionsListId,
+      iconName = 'Cancel',
```

Comment:
You shouldn't default this value here; let SuggestionsItem provide the default internally.